### PR TITLE
fixed broken layout for popular and simialr listing category

### PR DESCRIPTION
**Pull request by M4hd1BD**
Fixed this: https://ibb.co/PN4VwK7 to this: https://ibb.co/pKfD2tC

### CODE REVIEW

File: templates/widgets/featured-listing.php [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <?php
+
 /**
  * @author  wpWax
  * @since   7.3.0
  * @version 7.4.0
  */
 
-if ( ! defined( 'ABSPATH' ) ) exit;
+if (!defined('ABSPATH')) exit;
```
Comment (kowsar89) [inline templates/widgets/featured-listing.php:9]: Unnecessary space changes like this in all php files

**Review by kowsar89 — CHANGES_REQUESTED** [review]
php notice http://prntscr.com/hCyem8xkGwfn

File: templates/widgets/featured-listing.php (hunk 1 above)
Comment (M4hd1BD) [reply templates/widgets/featured-listing.php:9]: Fixed.

**Review by kowsar89 — CHANGES_REQUESTED** [review]
1. Fix php notice http://prntscr.com/hCyem8xkGwfn
2. Use correct node/npm version when building: http://prntscr.com/Wru19-JzxcAJ
3. Update template version number
4. Sync with alpha

**Review by kowsar89 — CHANGES_REQUESTED** [review]
don't change icon http://prntscr.com/s-n52gs09lxN

**Review by kowsar89 — CHANGES_REQUESTED** [review]
Few changes: http://prntscr.com/U_fjiOttBMj3

**Comment by M4hd1BD** [comment]
> Few changes: http://prntscr.com/U_fjiOttBMj3

Adjusted.

**Review by devShamim — CHANGES_REQUESTED** [review]
need to test with different themes
https://prnt.sc/Py1TyBoPyV_s

**Comment by M4hd1BD** [comment]
> need to test with different themes https://prnt.sc/Py1TyBoPyV_s

Improved.

**Review by devShamim — DISMISSED** [review]
Looks good!

**Review by kowsar89 — CHANGES_REQUESTED** [review]
Few style adjustment: http://prntscr.com/tYz0ZmKwm7Sw

**Review by kowsar89 — CHANGES_REQUESTED** [review]
Theme: Twenty Twenty-Three
Style broken: http://prntscr.com/ULsYd4C3aVzT

**Review by kowsar89 — APPROVED** [review]

**Pull request merged**